.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,41 +1,60 @@
 ## PR Summary
 
-## PR Checklist
-
-- [ ] Has Pytest style unit tests
-- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
-- [ ] New features are documented, with examples if plot related
-- [ ] Documentation is sphinx and numpydoc compliant
-- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
-- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way
-
 <!--
-Thank you so much for your PR!  To help us review your contribution, please
```

Comment:
What do you feel about leaving this nicety in?

Reply:
I actually would be in favor. I think the welcoming tone adds a lot, I just thought you guys were suggesting it was unnecessary :)

Reply:
Oops...yes please return it. 😃